src/main/java/Duke.java

```diff
@@ -37,342 +48,367 @@ public void run() throws DukeException {
+         */
+        void taskRemove(String nextLine) throws DukeException {
+            if (nextLine.equals("remove") || nextLine.equals("remove ")) {
+                throw new DukeException("OOPS!!! The description of remove cannot be empty");
             }
-
-            void taskEvent(String nextLine) throws DukeException {
-                if (nextLine.equals("event") || nextLine.equals("event ")) {
-                    throw new DukeException("OOPS!!! The description of an event cannot be empty.");
-                }
-                int charLoc = nextLine.indexOf("/at");
-                Event event = new Event(nextLine.substring(6, charLoc), nextLine.substring(charLoc + 4));
-                inputs.add(event);
-                int count = inputs.size();
-                System.out.println("Got it. I've added this task: \n" + "  [E][x] " + event.content +
-                        event.printedTime + "\n Now you have " + count + " tasks in the list");
-                storage.writeToFile(inputs);
+            int numTaskDone = Integer.valueOf(nextLine.substring(7));
+            if (numTaskDone > inputs.size()) {
+                throw new DukeException("OOPS!!! Task does not exist.");
             }
+            System.out.println("Noted. I've removed this task:");
+            Input inputType = inputs.get(numTaskDone - 1);
+            if (inputType.done) {
+                System.out.println("  " + inputType.id + "[/] " + inputType.content + inputType.printTime);
+            } else {
+                System.out.println("  " + inputType.id + "[x] " + inputType.content + inputType.printTime);
+            }
+            inputs.remove(numTaskDone - 1);
+            System.out.println("Now you have " + inputs.size() + " tasks in the list.");
+            storage.writeToFile(inputs);
         }
-            
-        public static class UI {
-            Scanner sc = new Scanner(System.in);
-
-            UI() {
-                String logo = " ____        _        \n"
-                        + "|  _ \\ _   _| | _____ \n"
-                        + "| | | | | | | |/ / _ \\\n"
-                        + "| |_| | |_| |   <  __/\n"
-                        + "|____/ \\__,_|_|\\_\\___|\n";
-                System.out.println("Hello from\n" + logo);
-                System.out.println("Hello! I'm Duke \n"
-                        + "What can I do for you?");
+
+        /*
+         * Adds a to-do object to the list, and saves the list to hard disk
+         *
+         * @param nextLine, represents user input
+         * @throws DukeException if user does not specify task to-do
+         */
+        void taskTodo(String nextLine) throws DukeException {
+            if (nextLine.equals("todo") || nextLine.equals("todo ")) {
+                throw new DukeException("OOPS!!! The description of a todo cannot be empty.");
             }
+            Todo todo = new Todo(nextLine.substring(5));
```

Comment:
I realised the use of substring to get specific keywords from the user input. An alternative to this would be to use the split() function of String to split the user input up!